.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+# Notes (delete this section)
+
+* For new APIs, follow http://aka.ms/apireview. This requires filing an issue
+  that outlines the new API first. Before we accept PRs, it muse be reviewed by
+  us and marked as `api-approved`.
+
+* Do not create tracking PRs with pending work. Submit the PR when you're ready
+  to review the changes with us. If you want to share work-in-progress PRs,
+  create them in your own fork.
+
+* Include a "Fixes #1 and #2" to link to the issue(s) this PR addresses.
+
+* Include a summary of the problem and the change so that reviewers don't have
+  to extract that information from the commit messages.
```

Comment:
Agreed on both accounts. I can actually see them becoming annoying but I'm willing to give a try.